Dockerfile

```diff
@@ -1,17 +1,17 @@
-ARG GO_VERSION=1.12.10
+ARG GO_VERSION=1.13.0
 
 # OS-X SDK parameters
 # NOTE: when changing version here, make sure to also change OSX_CODENAME below to match
-ARG OSX_SDK=MacOSX10.10.sdk
-ARG OSX_SDK_SUM=631b4144c6bf75bf7a4d480d685a9b5bda10ee8d03dbf0db829391e2ef858789
+ARG OSX_SDK=MacOSX10.11.sdk
+ARG OSX_SDK_SUM=98cdd56e0f6c1f9e1af25e11dd93d2e7d306a4aa50430a2bc6bc083ac67efbb8
```

Comment:
Once the SDK uploads to s3.dockerproject.org, I can change here. (If you want to re-generate it)